python/kvikio/_lib/nvcomp_cxx_api.pxd

```diff
@@ -73,7 +73,6 @@ cdef extern from "nvcomp/nvcompManager.hpp" namespace 'nvcomp':
             size_t uncompressed_buffer_size) except +
         nvcompStatus_t* get_status() const
         CompressionConfig(CompressionConfig& other)
-        CompressionConfig& operator=(CompressionConfig&& other) except +
```

Comment:
why are these removed?

Reply:
Cython doesn't actually support rvalue reference parameters. It never did, but previously it wouldn't warn about it. Now it does.

Note that despite this, you can still get the desired behavior by writing Cython code with an explicit move `x ==move(CompressionConfig())`. Cython will generate the C++ code assuming it'll use one of the other operators for an lvalue, but then the C++ compiler will use the rvalue version instead.

Reply:
The only issue might be that there's zero visibility of the rvalue versions on the Cython side. Should we leave a comment that a move is applicable when calling operator=?

Reply:
We could do that. We may need a more comprehensive strategy for this though since it's a problem that affects all of RAPIDS. There are multiple cases where we have functions that accept rvalues that we may want to leverage in Cython even though Cython doesn't think it knows how. @shwina do you have thoughts on how best to handle this?

Reply:
I don't have any fantastic suggestions. Cython doesn't really know about rvalues generally and getting it to generate the right code involves using explicit `move()` calls.

Agree with @vuule - perhaps we just leave a comment?

Reply:
Proposing such a change below. Maybe we can iterate on that change?

Reply:
Done.